client/src/use/useTrackFilters.ts

```diff
@@ -132,19 +132,18 @@ export default function useFilteredTracks(
     }
     markChangesPending();
   }
-  function updateTypeName({ currentType, newType }: {currentType: string; newType: string}) {
+  function updateTypeName({ currentType, newType }: { currentType: string; newType: string }) {
```

Comment:
Should there me something added on here that interacts and changes the `confidenceFilters` for the same existing type?
I.E. type `fish` has confidence filter set to 0.25, I rename type `fish` to `fish2` should this reset the confidenceFilter or should it stay the same?

Reply:
Only if the new type name doesn't already exist, I think.